packages/firestore/src/local/shared_client_state.ts

```diff
@@ -633,6 +645,16 @@ export class WebStorageSharedClientState implements SharedClientState {
     this.started = true;
   }
 
+  setSequenceNumberListener(
+    cb: (sequenceNumber: ListenSequenceNumber) => void
+  ): void {
+    this.sequenceNumberListener = cb;
+  }
+
+  writeSequenceNumber(sequenceNumber: ListenSequenceNumber): void {
+    this.setItem(SEQUENCE_NUMBER_KEY, JSON.stringify(sequenceNumber));
```

Comment:
JSON.stringify(sequenceNumber) -> String(sequenceNumber) ?

Reply:
We use `JSON.parse`, presumably we should use `JSON.stringify` to ensure it's compatible.